sdk/recoveryservices/Azure.ResourceManager.RecoveryServices/api/Azure.ResourceManager.RecoveryServices.netstandard2.0.cs

```diff
@@ -2,6 +2,8 @@ namespace Azure.ResourceManager.RecoveryServices
 {
     public static partial class RecoveryServicesExtensions
     {
+        public static Azure.Response<Azure.ResourceManager.RecoveryServices.Models.CapabilitiesResponse> CapabilitiesRecoveryService(this Azure.ResourceManager.Resources.SubscriptionResource subscriptionResource, Azure.Core.AzureLocation location, Azure.ResourceManager.RecoveryServices.Models.ResourceCapabilities input, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```

Comment:
rename to `GetCapabilitiesRecoveryService`

Reply:
this is probably due to operation ID: https://github.com/Azure/azure-rest-api-specs/blob/e217fd77475710b4a48542f433e3b9782f26f229/specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/stable/2023-01-01/vaults.json#L193

You can use `override-operation-name` directive to fix that. For example, see [here](https://github.com/Azure/azure-sdk-for-net/blob/ac35990e2a5320392fb09487687159273d88da99/sdk/compute/Azure.ResourceManager.Compute/src/autorest.md?plain=1#L75).

Reply:
@archerzz  this looks like a post call and not a get call. in case of POST what should be the name ?